manifest.yml.j2

```diff
@@ -57,6 +57,21 @@
       'production': 1
     },
   },
+  'notify-delivery-worker-ecs-fixup': {
+    'disk_quota': '10G',
+    'memory': '4G',
+    'additional_env_vars': {
+      'NOTIFICATION_QUEUE_PREFIX': NOTIFICATION_QUEUE_PREFIX + '-ecs',
+      'CELERYD_PREFETCH_MULTIPLIER': 1,
+      'CBC_PROXY_AWS_ACCESS_KEY_ID': CBC_PROXY_AWS_ACCESS_KEY_ID,
+      'CBC_PROXY_AWS_SECRET_ACCESS_KEY': CBC_PROXY_AWS_SECRET_ACCESS_KEY,
```

Comment:
You can get rid of these 2 lines. I can see these have come from the `notify-delivery-worker-broadcasts` which is actually an app no longer in use that is ready to be deleted! We aren't running any tasks through that app anymore as it was all to do with emergency alerts.

Reply:
gone